terraform/functions.tf

```diff
@@ -286,54 +291,130 @@ module "lambda_function-excelToJson" {
   source  = "terraform-aws-modules/lambda/aws"
   version = "6.5.0"
 
-  function_name = "excel-to-json"
+  // Metadata
+  function_name = "${var.namespace}-excelToJson"
   description   = "Reacts to S3 events and converts Excel files to JSON."
 
-  vpc_subnet_ids = local.private_subnet_ids
-  vpc_security_group_ids = [
-    module.lambda_security_group.id,
-    module.postgres.security_group_id,
-  ]
```

Comment:
Since the `excelToJson` handler doesn't seem to require DB access at the moment, we can avoid adding the Lambda function to the VPC.